.github/workflows/prototyping.yml

```diff
@@ -0,0 +1,55 @@
+name: prototyping
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'prototyping'
+    paths:
+      - '.github/workflows/**'
+      - 'prototyping/**'
+  pull_request:
+    paths:
+      - '.github/workflows/**'
+      - 'prototyping/**'
+
+jobs:
+  linux:
+    strategy:
+      matrix:
+        agda: [2.6.2.1]
+    runs-on: ubuntu-latest
```

Comment:
Adding

    name: prototyping

here would clean up the auto-generated build name